src/h3/api/_api_template.py

```diff
@@ -1,20 +1,20 @@
-import h3py.h3core as h3core
-import h3py.util as u
+import h3.h3core as h3core
+import h3.h3utils as u
```

Comment:
why the change from `util` to `h3utils`?

Reply:
`util` seemed like a pretty generic name, I wanted to disambiguate to protect from potential PyPi clashes(eg: [utils package](https://pypi.org/project/utils/), [util package (defunct)](https://pypi.org/project/util/)) while doing `import util` from the h3 folder

Reply:
fair enough!